Makefile-az.mk

```diff
@@ -25,6 +25,32 @@ az-all-custom-vnet: az-login az-create-workload-msi az-mkaks-custom-vnet az-crea
 
 az-all-savm:    az-login az-mkaks-savm az-perm-savm az-patch-skaffold-azure az-build az-run az-run-sample ## Provision the infra (ACR,AKS); build and deploy Karpenter; deploy sample Provisioner and workload - StandaloneVirtualMachines
 
+install-tools: az-tool ## install tools to create a local developer environment
+	sudo apt update && sudo apt install curl jq -y
+	curl -Lo yq https://github.com/mikefarah/yq/releases/download/v4.43.1/yq_linux_amd64 && chmod +x yq && sudo mv yq /usr/local/bin
+	if test -z "$(shell which helm)"; then \
+		curl -sL https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3 | bash -
```

Comment:
For any of the tools that provide an alternative to piping into bash - please use the alternative.